beetmoverscript/Dockerfile.test

```
@@ -7,7 +7,8 @@ WORKDIR /app
 COPY MANIFEST.in setup.py tox.ini /app/
 COPY requirements/ /app/requirements/
 
-RUN pip install -r requirements/local.txt
+ARG PYTHON_REQ_SUFFIX
```

Nit: We can define a default value so that we don't have to pass `--build-arg PYTHON_REQ_SUFFIX=      ` on python 3.8. 

```suggestion
ARG PYTHON_REQ_SUFFIX=''
```

---

Oh looks like `ARG PYTHON_REQ_SUFFIX` already sets it to an empty string. TIL!